components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/store/JDBCRecoveryDataStore.java

```diff
@@ -292,16 +292,59 @@ public UserRecoveryData loadWithoutCodeExpiryValidation(User user) throws Identi
         return null;
     }
 
-
     @Override
-    public void invalidate(User user) throws IdentityRecoveryException {
+    public UserRecoveryData loadWithoutCodeExpiryValidation(User user, Enum recoveryScenario)
+            throws IdentityRecoveryException {
 
         PreparedStatement prepStmt = null;
-        Connection connection = IdentityDatabaseUtil.getDBConnection(true);
+        ResultSet resultSet = null;
+        Connection connection = IdentityDatabaseUtil.getDBConnection(false);
+
```

Comment:
Remove unnecessary new lines.

Reply:
Addressed in https://github.com/wso2-extensions/identity-governance/pull/414/commits/1f1503b13f90b814eca821c6fca73b9af4c61052